### PR TITLE
Repackage as a non-namespace package.

### PR DESCRIPTION
**Pull request by dave-shawley**
This PR moves to using the non-namespace package `sprockets_amqp` as a replacement for `sprockets.mixins.amqp`.  Both packages are distributed in the Python distributable and the namespace-based package contains the necessary compatibility imports to make the change seamless (I think).

The rationale for this change is to remove the usage of [namespace packages](http://setuptools.readthedocs.io/en/latest/setuptools.html#namespace-packages) from the sprockets universe.  In hindsight, they haven't provided benefits commensurate with their headaches -- deeply nested packages, broken mixture of py_module vs package, bundling everything in _\_\_init\_\_.py_, etc. etc.  The structure of the resulting package is flat and much simpler IMO.

The next step will be to move `sprockets_amqp` into it's own repository and Python distributable and mark this package as deprecated.

### CODE REVIEW

File: sprockets_amqp/client.py [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+    If RabbitMQ closes the connection, it will reopen it. You should
+    look at the output, as there are limited reasons why the connection may
+    be closed, which usually are tied to permission related issues or
+    socket timeouts.
+
+    If the channel is closed, it will indicate a problem with one of the
+    commands that were issued and that should surface in the output as well.
+
+    """
+    STATE_DESC = STATE_DESC
```
Comment (gmr) [inline sprockets_amqp/client.py]: I don't think anything is using this but the class for ``self.state_description`` -- why keep it?

**Review by gmr — APPROVED** [review]

File: sprockets_amqp/client.py (hunk 1 above)
Comment (gmr) [reply sprockets_amqp/client.py]: @dave-shawley should I merge at this point?

Comment (dave-shawley) [reply sprockets_amqp/client.py]: Bah ... I missed that one. lemme fix it

**Comment by dave-shawley** [comment]
@gmr - make sure that this works with some of our services.  I haven't tested that I maintained compatibility with anything "real".  My intent was compatibility so we shouldn't need a major version bump.

**Comment by rklingsberg** [comment]
@dave-shawley I tested locally with Tagging, unit tests and sending a test request through the API both worked without issue.

**Review by rklingsberg — APPROVED** [review]

**Comment by dave-shawley** [comment]
this is too old to be useful from my perspective

**Pull request closed without merging**